.all-contributorsrc

```diff
@@ -617,6 +617,15 @@
       "contributions": [
         "content"
       ]
+    },
+    {
+      "login": "melanieganz",
+      "name": "melanieganz",
```

Comment:
@melanieganz Can I use your full name to acknowledge the contribution? (Currently, it would only list you under your github user name.)

Reply:
I will merge this now as is, but please let me know at any point if I can put in your name, too. :)